server/aws-lsp-codewhisperer/src/language-server/netTransformServer.ts

```diff
@@ -92,14 +96,17 @@ export const QNetTransformServerToken =
                     }
                     case PollTransformForPlanCommand: {
                         const request = params as GetTransformRequest
-                        logging.log('Calling pollTransformForPlan request with job Id: ' + request.TransformationJobId)
+                        logging.log(
+                            'Calling pollTransformForPlan request with Transformation Job ID: ' +
+                                request.TransformationJobId
+                        )
```

Comment:
Taking an opinionated stance: we should only mention names of API endpoints in these logs. Naming an internal function will: 
1. Not have meaning for the customer
2. Cause confusion for ourselves (from just the logs, how will we know if we're talking about an actual endpoint or some function? Did we mean to reference an actual endpoint and it was misspelled?)

Suggestion: 
"Awaiting  transformationPlan for Transformation Job ID: ${request.TransformationJobId}."